entity-framework/core/modeling/relationships/one-to-one.md

```diff
@@ -0,0 +1,682 @@
+﻿---
+title: One-to-one relationships - EF Core
+description: How to configure one-to-one relationships between entity types when using Entity Framework Core
+author: ajcvickers
+ms.date: 02/25/2023
+uid: core/modeling/relationships/one-to-one
+---
+# One-to-one relationships
+
+One-to-one relationships are used when one entity is associated with at most one other entity. For example, a `Blog` has one `BlogHeader`, and that `BlogHeader` belongs to a single `Blog`.
+
+A one-to-one relationship is made up from:
+
+- One or more [primary or alternate key](xref:core/modeling/relationships/foreign-and-principal-keys#principal-keys) properties on the principal entity. For example, `Blog.Id`.
+- One or more [foreign key](xref:core/modeling/relationships/foreign-and-principal-keys#foreign-keys) properties on the dependent entity. For example, `BlogHeader.BlogId`.
+- Optionally, a [reference navigation](xref:core/modeling/relationships/navigations#reference-navigations) on the principal entity referencing the dependent entity. For example, `Blog.Header`.
+- Optionally, a [reference navigation](xref:core/modeling/relationships/navigations#reference-navigations) on the dependent entity referencing the principal entity. For example, `BlogHeader.Blog`.
+
+It is not always obvious which side of a one-to-one relationship should be the principal, and which side should be the dependent. Some considerations are:
```

Comment:
I disagree. This is one of the most difficult things that people struggle with in 1:1 relationships. Its well worth understanding this and thinking about it up front to avoid problems down the line.